x-pack/plugins/global_search_bar/public/telemetry/telmetry.test.tsx

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ChromeStyle } from '@kbn/core-chrome-browser';
+import { applicationServiceMock, coreMock } from '@kbn/core/public/mocks';
+import { GlobalSearchBatchedResults, GlobalSearchResult } from '@kbn/global-search-plugin/public';
+import { globalSearchPluginMock } from '@kbn/global-search-plugin/public/mocks';
+import { __IntlProvider as IntlProvider } from '@kbn/i18n-react';
+import { usageCollectionPluginMock } from '@kbn/usage-collection-plugin/public/mocks';
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { of, throwError } from 'rxjs';
+import { EventReporter } from '.';
+import { SearchBar } from '../components/search_bar';
+
+jest.mock(
+  'react-virtualized-auto-sizer',
+  () =>
+    ({ children }: any) =>
+      children({ height: 600, width: 600 })
+);
+
+type Result = { id: string; type: string } | string;
+
+const createResult = (result: Result): GlobalSearchResult => {
+  const id = typeof result === 'string' ? result : result.id;
+  const type = typeof result === 'string' ? 'application' : result.type;
+  const meta = type === 'application' ? { categoryLabel: 'Kibana' } : { categoryLabel: null };
+
+  return {
+    id,
+    type,
+    title: id,
+    url: `/app/test/${id}`,
+    score: 42,
+    meta,
+  };
+};
+
+const createBatch = (...results: Result[]): GlobalSearchBatchedResults => ({
+  results: results.map(createResult),
+});
+jest.useFakeTimers({ legacyFakeTimers: true });
```

Comment:
I never know if this affects other tests 😊  So I put it in a `beforeAll` and then in an `afterAll` I call `jest. useRealTimers()` ... just in case